src/codegen/templates/_task_modules/channels/ai_channel.py.mako

```diff
@@ -0,0 +1,40 @@
+<%
+    from codegen.utilities.attribute_helpers import get_attributes, get_enums_used
+    from codegen.utilities.text_wrappers import wrap
+    attributes = get_attributes(data, "AIChannel")
+    enums_used = get_enums_used(attributes)
+%>\
+# Do not edit this file; it was automatically generated.
+
+import ctypes
+import numpy
+import deprecation
+import __init__
+
+from nidaqmx._lib import (
+    lib_importer, wrapped_ndpointer, ctypes_byte_str, c_bool32)
+from nidaqmx.scale import Scale
+from nidaqmx.errors import (
+    check_for_error, is_string_buffer_too_small, is_array_buffer_too_small)
+from nidaqmx._task_modules.channels.channel import Channel
+from nidaqmx.utils import unflatten_channel_string
+from nidaqmx.constants import (
+    ${', '.join([c for c in enums_used]) | wrap(4, 4)})
+
+__version__ = __init__.__version__
```

Comment:
`__version__` is not required. and copying the value from `__init__` seems like overkill.

Can you help me understand why we're doing this?

Reply:
We're doing this to pass the current version to the `deprecation` module. Perhaps we shouldn't bother? I think it will still print the `deprecated_in` version without `current_version`.

Reply:
Getting the version from `__init__` isn't working anyway. I think we would need to introduce a new submodule that is not named `__init__.py` to make this work.

Since we're only using `deprecated_in` right now, not `removed_in`, I think it would be simpler to remove `current_version=`.